docker/xdebug.dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ${DOCKERIZE_BASE_IMAGE}
+
+#
+ENV APP_VERSION="${DOCKERIZE_VERSION}"
+
+#
+ENV DOCKERIZE_VERSION="${DOCKERIZE_VERSION}"
+ENV DOCKERIZE_BRANCH="${DOCKERIZE_BRANCH}"
+ENV DOCKERIZE_COMMIT="${DOCKERIZE_COMMIT}"
+
+USER root
+
+RUN pecl install xdebug-2.9.8 \
+    && docker-php-ext-enable xdebug \
+    && echo "xdebug.remote_host = 10.254.254.254" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
```

Comment:
Woher kommt die IP-Adresse?

Du kannst die DNS `host.docker.internal` angeben, über die Container immer den Host erreichen (https://docs.docker.com/docker-for-windows/networking/).

Reply:
👍